### DISCUSSION TITLE
Mark duplicate IP addresses

### DISCUSSION THREAD

**Discussion by Abdalrhmanalfalaha** [netbox-community/netbox#10582] [Ideas]
Hello, can we mark duplicate ip addresses with color e.g. "red" or create boolean field (duplicated: true or false) for ip addresses? So that we can always direct see if an IP address is duplicated or not? It would also be very good if we get a warning message when creating IP addresses that this IP is already created.

Thanks!

**Comment by candlerb** [comment]
> It would also be very good if we get a warning message when creating IP addresses that this IP is already created.

What version of Netbox are you using?

You can set `ENFORCE_GLOBAL_UNIQUE = True` in configuration.py - this will prevent them being created in the first place, except under race conditions (i.e. when two users or scripts create the same IP at exactly the same instant).

When you view an individual IP address, it does already flag that there are duplicates (in red). I just tested this on https://demo.netbox.dev/

![image](https://user-images.githubusercontent.com/44789/194698504-1959e167-126a-4c24-8eb5-e0ad0a382b32.png)

**Comment by Abdalrhmanalfalaha** [comment]
It works! Thank you!
